gradle/gradle-mvn-push.gradle

```diff
@@ -0,0 +1,142 @@
+/*
```

Comment:
Is there a way in which we can have this and other files that are the same as the ones in arrow in a single place? This is gonna be an increasing issue with all other libs splitting out of the monorepo

Reply:
Thanks @raulraja for the idea, done! 
We cannot use the original repository because we adapted it for Arrow, so I changed it to use the file from Arrow repository. 
In case we have more repositories, we could move it to a single repository.